terraform-tests/aurora_postgresql_test.go

```diff
@@ -101,6 +102,7 @@ var _ = Describe("Aurora postgresql", Label("aurora-postgresql-terraform"), Orde
 				"preferred_backup_window":            Equal("23:26-23:56"),
 				"copy_tags_to_snapshot":              BeTrue(),
 				"deletion_protection":                BeFalse(),
+				"engine_version":                     Equal("8.0.postgresql_aurora.3.02.0"),
```

Comment:
**question**: Since we're adding this, can we assign a value that is related to Postgres?

Reply:
Done!! Thank you very much @jhvhs